tools/pkglint/lib/rules.ts

```diff
@@ -1580,5 +1594,5 @@ function toRegExp(str: string): RegExp {
 }
 
 function readBannerFile(file: string): string {
-  return fs.readFileSync(path.join(__dirname, 'banners', file), { encoding: 'utf-8' });
+  return fs.readFileSync(path.join(__dirname, 'banners', file), { encoding: 'utf-8' }).trim();
```

Comment:
This `trim()` ensures we don't insert too much vertical space in the stability banners, as `markdownlint` would subsequently consider those are formatting errors.

Reply:
Consider adding this comment directly into the code.

Reply:
Consider adding this comment directly into the code.